.travis.yml

```diff
@@ -1,22 +1,30 @@
 language: node_js
 
 node_js:
-  - "4"
-  - "6"
-  - "8"
-
+  - '4'
+  - '6'
+  - '8'
 sudo: false
 
 script:
- - npm test
+  - npm test
 
 notifications:
   irc:
     channels:
-      - "chat.freenode.net#datbots"
+      - chat.freenode.net#datbots
     template:
-      - "%{repository_slug} - %{commit_subject} - %{result} - %{build_url}"
+      - '%{repository_slug} - %{commit_subject} - %{result} - %{build_url}'
     skip_join: true
     on_success: change
     on_failure: always
 
+deploy:
```

Comment:
trying to figure it out =)

1. package binary with `pkg` (`npm run package`)
2. travis automatically builds after tests pass
3. uploads to github releases

I don't see the result of this last build anywhere though. Not sure how it works without the tags.